### DISCUSSION TITLE
what will happen if typescript decides to turn-off the experimental decorator feature in their new release??

### DISCUSSION THREAD

**Discussion by alok2k5ingh** [MichalLytek/type-graphql#1291] [Q&A] [answered]
what will happen if typescript decides to turn-off the experimental decorator feature in their new release?? @MichalLytek

**Comment by MichalLytek** [comment, marked as answer]
There's nothing to be afraid of. It's not the only library that uses TS types metadata. We can replace it even now with other compiler plugins.
